spec/shared/helpers/application_helper/buttons/basic.rb

```diff
@@ -0,0 +1,25 @@
+shared_context 'ApplicationHelper::Button::Basic' do
+  let(:view_context) { setup_view_context_with_sandbox(sandbox) }
+  let(:button) { described_class.new(view_context, {}, instance_data, props) }
+  subject { button }
+end
+
+shared_examples 'ApplicationHelper::Button::Basic#visible?' do |visible|
```

Comment:
Can't be this split into `visible` and `not visible` or `invisible`, same as the `enabled/disabled`?